### PR TITLE
Modificações efetuadas em: gitahead_pt_BR.ts

### PR DESCRIPTION
**Pull request by gomes2191**
New translations have been made.

### CODE REVIEW

File: l10n/gitahead_pt_BR.ts [hunk 1]
```diff
@@ -3540,48 +3581,55 @@ Exemplos
...
-        <translation type="unfinished">Rebase...</translation>
+        <translatorcomment>Pega os commits do branch pai, sincroniza com os commits anterior desse, e joga os commits do branch atual no final da pilha.</translatorcomment>
+        <translation>Rebase...</translation>
     </message>
     <message>
         <location filename="../src/ui/ReferenceView.cpp" line="569"/>
         <source>Squash...</source>
-        <translation type="unfinished">Merge --squash...</translation>
+        <translatorcomment>Mescla alocando as alterações do Branch especificado no topo, mesclando como um novo commit.</translatorcomment>
+        <translation>Merge --Squash...</translation>
```
Comment (Philipp91) [inline l10n/gitahead_pt_BR.ts:3614]: This looks wrong.

File: l10n/gitahead_pt_BR.ts [hunk 2]
```diff
@@ -3540,48 +3581,55 @@ Exemplos
...
         <source>Rebase...</source>
-        <translation type="unfinished">Rebase...</translation>
+        <translatorcomment>Pega os commits do branch pai, sincroniza com os commits anterior desse, e joga os commits do branch atual no final da pilha.</translatorcomment>
+        <translation>Rebase...</translation>
     </message>
     <message>
         <location filename="../src/ui/ReferenceView.cpp" line="569"/>
         <source>Squash...</source>
-        <translation type="unfinished">Merge --squash...</translation>
+        <translatorcomment>Mescla alocando as alterações do Branch especificado no topo, mesclando como um novo commit.</translatorcomment>
```
Comment (Philipp91) [inline l10n/gitahead_pt_BR.ts:3613]: Is this translatorcomment displayed anywhere? It seems like you're filling this in like a tooltip that's displayed to the user, but I believe it's more like documentation for future developers who might change/update the translation.

File: l10n/gitahead_pt_BR.ts [hunk 3]
```diff
@@ -3344,112 +3363,130 @@ Exemplos
         <location filename="../src/ui/PathspecWidget.cpp" line="115"/>
         <source>Filter by Path</source>
         <translatorcomment>FIltra por diretório...</translatorcomment>
-        <translation>FIltra por diretório</translation>
+        <translation>FIltrar por Diretório</translation>
```
Comment (Philipp91) [inline l10n/gitahead_pt_BR.ts:3366]: I think the `I` should be lower-case.

**Review by Philipp91 — COMMENTED** [review]
Thanks! Just a few minor comments.

File: l10n/gitahead_pt_BR.ts (hunk 2 above)
Comment (gomes2191) [reply l10n/gitahead_pt_BR.ts:3613]: Basically I inserted this comment to specify what each action does, currently the application does not have this explanatory feature that triggers the comment when you hover the mouse pointer over some action, I placed the observation just to facilitate the translation process, but in the future is a resource to think about...

**Pull request merged**